april_tag_updates/docking_script.py

```diff
@@ -0,0 +1,127 @@
+expected_dist = 37
+known_width = 4.18 ##inches
+focal_length = 1.37478492e+03
+from electrical.motor_controller import BasicMotorController
+import numpy as np
+import cv2
+import time 
+from enum import Enum
+
+arucoDict = cv2.aruco.Dictionary_get(cv2.aruco.DICT_APRILTAG_36h11)
+arucoParams = cv2.aruco.DetectorParameters_create()
+cap = cv2.VideoCapture(0)
+window = 'Camera'
+cv2.namedWindow(window)
+def distance_to_camera(knownWidth, focalLength, perWidth):
+  # compute and return the distance from the maker to the camera
+  return (knownWidth * focalLength) / perWidth
+
+def direction_of_tag(corner,image):
+  (_, _, _, _) = corner
+  _,w = image.shape[0], image.shape[1]
+  cx, _ = int(960),int(540)
```

Comment:
is this based on how large the window is? would it be more robust to find the width and length of the window and dividing that by 2?

Reply:
We know that the camera has 1080p resolution which means we already know the image has 1920 pixels horizontally by 1080 vertically. The center of the image is dividing these numbers by 2. It's given info that we can add to documentation on this file.

Reply:
yeah, I think it'll make sense to add to documentation

Reply:
because theres no documentation, i dont know what cx is and i dont really understand the relation that this has to bottomLeft and determining right or left.

does right/left signify that we need to turn right/turn left?